### DISCUSSION TITLE
Feature request: Adding custom code snippets to BG Script without sync to VSCode

### DISCUSSION THREAD

**Discussion by Zmud1** [arnoudkooi/SN-Utils#266] [Ideas]
Hi,
Is there any possibility to allow the user to add custom code snippets to BG Script editor directly, without synchronization with VS Code? 

Certainly, many developers have their own snippets created that speed up their work. I do realize that we can sync the BG Code editor with VS Code, and then use our snippets in VS so that the code is reflected in BG script. However I find it pretty annoying that just to use that feature I need to have to opened BG Script and VS Code in different windows and switch between them. 

Example of custom snippet in VS Code:
![image](https://user-images.githubusercontent.com/43832632/179014566-e41cf005-e513-45b3-a3dd-c86789a00ea4.png)
![image](https://user-images.githubusercontent.com/43832632/179014648-dc8aa776-0184-4f76-aaed-909a6f3b0abf.png)

While I am not able to use it in BG Script editor:
![image](https://user-images.githubusercontent.com/43832632/179014816-eb15380b-9abc-417b-be42-1115a371ad87.png)

**Comment by arnoudkooi** [comment]
This is one for the longer term backlog :) 
I want to make this available both for background script as well as other places.
I'll move it to a discussion for now, and will make it an issue again once there is activity.

**Reply by Zmud1** [reply]
I am waiting with anticipation :)
